src/Teams/Models/MultipleQuestionEditModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Teams.Models
+{
+    public class MultipleQuestionEditModel
```

Comment:
The observations from the MultipleQuestionAddModel model applicable for this model too.